core/src/main/java/org/frankframework/receivers/PullingListenerContainer.java

```diff
@@ -89,6 +96,12 @@ public void configure() {
 			}
 			txNew = txDef;
 		}
+
+		if (receiver.getListener() instanceof IPeekableListener<M>) {
+			messagePeekingStatistics = metricsInitializer.createDistributionSummary(receiver, FrankMeterType.LISTENER_MESSAGE_PEEKING);
```

Comment:
Als het goed is maakt de createSubDistributionSummary gebruik van de parent tags maar de listener name.
```suggestion
			messagePeekingStatistics = metricsInitializer.createSubDistributionSummary(receiver, receiver.getListener(), FrankMeterType.LISTENER_MESSAGE_PEEKING);
```
(en hier onder ook)

Reply:
Aangepast